app/containers/message/Reactions.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import Touchable from 'react-native-platform-touchable';
+import PropTypes from 'prop-types';
+
+import { CustomIcon } from '../../lib/Icons';
+import styles from './styles';
+import Emoji from './Emoji';
+import { BUTTON_HIT_SLOP } from './utils';
+
+const AddReaction = React.memo(({ toggleReactionPicker }) => (
+	<Touchable
+		onPress={toggleReactionPicker}
+		key='message-add-reaction'
+		testID='message-add-reaction'
+		style={styles.reactionButton}
+		background={Touchable.Ripple('#fff')}
+		hitSlop={BUTTON_HIT_SLOP}
+	>
+		<View style={styles.reactionContainer}>
+			<CustomIcon name='add-reaction' size={21} style={styles.addReaction} />
+		</View>
+	</Touchable>
+));
+
+const Reaction = React.memo(({
+	reaction, user, onReactionLongPress, onReactionPress, baseUrl, getCustomEmoji
+}) => {
+	const reacted = reaction.usernames.findIndex(item => item === user.username) !== -1;
```

Comment:
unnecessary `!== -1`

Reply:
It's is necessary.
I want to know if it IS found.
`findIndex` returns `-1` when not found.
`-1` is not falsey.